features/geometrisch-zoeken.feature

```diff
@@ -0,0 +1,700 @@
+# language: nl
+
+Functionaliteit: Als gebruiker wil ik met een geometrie (punt, bounding box, polygoon) kunnen zoeken naar resources, 
+                 zodat ik gericht vervolgstappen kan definiëren m.b.t. de gevonden resources.
+
+    De resources waarvan de geometrie tenminste één punt gemeenschappelijk hebben met de opgegeven geometrie worden geretourneerd.
+
```

Comment:
De toelichting hieronder over intersects is niet nodig toch? De essentie daarvan wordt toch al geraakt door bovenstaande zin?

Bovendien is het verwarrend om m.n. voor zoeken met bbox te zeggen: "wordt een intersects spatial operator gebruikt". Wat is in deze context een "operator"?

Reply:
Aangepast (ook in PR 443).